### ISSUE TITLE
Can not apply textures of fbx model

### ISSUE THREAD

**Issue by dementia2029** [google-ar/sceneform-android-sdk#270]
I have an FBX model and textures:
https://drive.google.com/file/d/1LI9qP-NnqVejJ7kRgyHHRA3ssu6ArPhZ/view?usp=sharing
When i try to setup textures in sfa file like this:
```
name: "cloth__",
        parameters: [
           {
              baseColorMap: "close_base",
           }
.......
samplers: [
         {
            file: "sampledata/models/cloth___Base_Color.jpg",
            name: "close_base",
            params: {
                usage_type: "Data"
            }
         }
```
Or like this:
```
 name: "wood",
         parameters: [
            {
               baseColorMap: "wood_base",
            }
......
{
            file: "sampledata/models/wood_Base_Color.png",
            name: "wood_base",
            params: {
                usage_type: "Color"
            }
         }
```
Textures does not apply.
But i am able to setup baseColor<vec4> and it works. Also, if i add pipeline_name then an error during building is occurred. Pls, any help. Thanks.

**Comment by dementia2029** [comment]
My latest sfa file:
[Divan.sfa.zip](https://github.com/google-ar/sceneform-android-sdk/files/2305358/Divan.sfa.zip)

**Comment by dementia2029** [comment]
Anyone?

**Comment by malik-at-work** [comment]
Sorrry for the long response time. We did track this issue as a bug and we resolved it in Sceneform 1.5  Please retest this.

**Comment by claywilkinson** [comment]
Please reopen this issue if you are still having issues with the latest Scenform SDK.
